python/openai/openai/src/schemas/openai.py

```diff
@@ -0,0 +1,867 @@
+# generated by fastapi-codegen:
+#   filename:  api-spec/openai_trimmed.yml
```

Comment:
Include the yaml instead and only code-gen at build time?

Reply:
I believe there were some manual edits needed - and not sure how easy it would be to seperate those into a seperate file . We should have clear instructions on the steps to generate and modify though - and reference the specific version of fastapi-codegen we used.

Reply:
The codegen tool produced a lot of older pydantic v1 concepts that are currently deprecated in pydantic v2, and will be removed in pydantic v3 - so I made manual edits there. 

Ideally we'd spend more time to polish and document the codegen path in a way that won't need manual edits - but I'm considering that lower priority for now. I think it would be a good task to take on when adding another endpoint such as embeddings.